fpm-entrypoint.sh

```diff
@@ -40,8 +40,9 @@ then
   PACKAGE_REPLACES=kong-community-edition
   PACKAGE_REPLACES_2=kong-enterprise-edition-fips
 
-elif [ "$KONG_PACKAGE_NAME" = "kong-enterprise-edition-fips" ]
+elif [ "$KONG_PACKAGE_NAME" = "kong-enterprise-edition-fips" ] || [ "$SSL_PROVIDER" = "boringssl" ]
```

Comment:
```suggestion
elif [ "$SSL_PROVIDER" = "boringssl" ]
```
(the first clause will never be true, unless the package name is already set, in which case it's unnecessary to set it again)

Reply:
`KONG_PACKAGE_NAME` [should](https://github.com/Kong/kong-build-tools/blob/master/Makefile#L40) always be set